include/nbl/builtin/hlsl/glsl_compat/core.hlsl

```diff
@@ -30,6 +39,12 @@ genIUType bitfieldInsert(genIUType const& Base, genIUType const& Insert, int Off
 	return glm::bitfieldInsert<genIUType>(Base, Insert, Offset, Bits);
 }
 
+template<typename T>
+inline int findMSB(T val)
+{
+    return hlsl::findMSB(val);
+}
+
```

Comment:
as per the discord convo, the `hlsl` namespace should sit above both `glsl` and `spirv`

Reply:
removed `glsl::findMSB`